cvat/apps/engine/migrations/0092_labeledimageattributeval_job_and_more.py

```diff
@@ -0,0 +1,127 @@
+# Generated by Django 4.2.21 on 2025-07-07 09:30
+
+import django.db.models.deletion
+from django.db import connection, migrations, models
+
+
+def set_attributeval_job_id(apps, schema_editor):
+    with connection.cursor() as cursor:
+        cursor.execute(
```

Comment:
Why we need a custom SQL for this? Can't it be implemented like here: https://github.com/cvat-ai/cvat/blob/7b9a7f60065ce099f4b4807da9df007fa26da315/cvat/apps/quality_control/migrations/0007_project_quality.py#L14-L28 via `Subquery` and `OuterRef`?